README.md

```diff
@@ -70,3 +70,29 @@ For example:
   🚀 My Custom App
 </span>
 ```
+
+### Theme Override
+To override the theme, create a `auth_style/extra_head.html` within your project's templates directory. You will then be able to define your own CSS variables to override defaults.
```

Comment:
Could you put this in a more normative / assertive style? Defining arbitrary "your own CSS variables" won't necessarily do anything; the *only* supported thing is to define new DaisyUI theme variables.

While it would be theoretically possible for a downstream to put other arbitrary CSS here, we can't support any stability of the structure of our upstream HTML, so I don't think we should mention it.

Reply:
Absolutely, makes sense. Update now!